apps/console/src/features/identity-providers/components/forms/authenticators/saml-authenticator-form.tsx

```diff
@@ -0,0 +1,546 @@
+                ) }
+                validate={ composeValidators(
+                    required,
+                    hasLength(SERVICE_PROVIDER_ENTITY_ID_LENGTH)
+                ) }
+                hint={ (
+                    <>
+                        This value will be used as the <Code>&lt;saml2:Issuer&gt;</Code> in
+                        the SAML requests initiated from { config.ui.productName } to external
+                        Identity Provider (IdP). You need to provide a unique value
+                        as the service provider <Code>entityId</Code>.
+                    </>
+                ) }
+            />
+
+            <Field.Input
+                required={ true }
+                name="SSOUrl"
+                value={ formValues?.SSOUrl }
+                inputType="default"
+                placeholder={ t(`${ I18N_TARGET_KEY }.SSOUrl.placeholder`) }
+                ariaLabel={ t(`${ I18N_TARGET_KEY }.SSOUrl.ariaLabel`) }
+                data-testid={ `${ testId }-SSOUrl-field` }
+                label={ (
+                    <FormInputLabel htmlFor="SSOUrl">
+                        { t(`${ I18N_TARGET_KEY }.SSOUrl.label`) }
+                    </FormInputLabel>
+                ) }
+                maxLength={ SSO_URL_LENGTH.max }
+                minLength={ SSO_URL_LENGTH.min }
+                validate={ composeValidators(
+                    required,
+                    isUrl,
+                    hasLength(SSO_URL_LENGTH)
+                ) }
+                hint={ t(`${ I18N_TARGET_KEY }.SSOUrl.hint`, {
+                    productName: config.ui.productName
+                }) }
+            />
+
+            <Field.Input
+                required={ true }
+                name="IdPEntityId"
+                value={ formValues?.IdPEntityId }
+                inputType="default"
+                placeholder={ t(`${ I18N_TARGET_KEY }.IdPEntityId.placeholder`) }
+                ariaLabel={ t(`${ I18N_TARGET_KEY }.IdPEntityId.ariaLabel`) }
+                data-testid={ `${ testId }-IdPEntityId-field` }
+                label={ (
+                    <FormInputLabel htmlFor="IdPEntityId" disabled={ true }>
+                        { t(`${ I18N_TARGET_KEY }.IdPEntityId.label`) }
+                    </FormInputLabel>
+                ) }
+                maxLength={ IDENTITY_PROVIDER_ENTITY_ID_LENGTH.max }
+                minLength={ IDENTITY_PROVIDER_ENTITY_ID_LENGTH.min }
+                validate={ composeValidators(
+                    required,
+                    hasLength(IDENTITY_PROVIDER_ENTITY_ID_LENGTH)
+                ) }
+                hint={ (
+                    <>
+                        This is the <Code>&lt;saml2:Issuer&gt;</Code> value specified in
+                        the SAML responses issued by the external IdP. Also, this needs to
+                        be a unique value to identify the external IdP within your
+                        organization.
```

Comment:
Please move to localisation if possible, check on other similar places as well.

Reply:
Tracking an issue for this. I will address them with https://github.com/wso2-enterprise/asgardeo-product/issues/5593